docs/product/data-management-settings/filtering/index.mdx

```diff
@@ -33,6 +33,14 @@ Legacy browser filters were updated in Feb 2024 and will be periodically evaluat
 
 If you had a legacy browser filter on before the update, the old filter will appear in your settings as "Deprecated". Deprecated legacy browser filters still work. However, if you turn them off, you won’t be able to turn them on again and will need to use the new filters instead.
 
+### Browser Extension Errors
+
+Some browser extensions are known to cause errors. You can use the browser extension filter to filter these out. The filter checks the error message and the event source to see if it's a known error coming from a browser extension.
+
+### Web Crawler Errors
+
+Due to their nature, web crawlers often encounter errors that normal users won't see. You can use the web crawler filter to filter out errors from web crawlers from the following sites: Baidu, Yahoo, Sogou, Facebook, Alexa, Slack, Google Indexing, Pingdom, Lytics, AWS Security Scanner, Hubspot, Bytedance, and other generic bots and spiders. Errors from Slack's Slackbot web crawler will not be filtered. 
```

Comment:
Is it worth mentioning that we use the user-agent to check these?

Reply:
yeah, let me add something in!